server/rest/src/main/java/org/infinispan/rest/resources/CacheManagerResource.java

```diff
@@ -196,21 +210,28 @@ public Invocations getInvocations() {
       NettyRestResponse.Builder responseBuilder = checkCacheManager(request);
       if (responseBuilder.getHttpStatus() == NOT_FOUND) return completedFuture(responseBuilder.build());
 
+      EmbeddedCacheManager withSubjectCM = cacheManager.withSubject(request.getSubject());
```

Comment:
withSubjectCM is not particularly pleasant. `subjectCacheManager` is nicer (like you do below)

Reply:
ok